### PR TITLE
Auto database update fixes

### PR DESCRIPTION
**Pull request by MattCloward**
Added:
- beta unit filtering script and optional parameter
- character normalizing script that runs right after the associations_table is first created
- small comment improvements on time required by scripts

### CODE REVIEW

File: update_database_scripts/normalizeChars.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# where: "associationTableFolderPath" is the path to the folder where the associations table is stored
+
+associationTableFolderPath = argv[1] if len(argv) > 1 else "../tables/"
+associationsTablePath = path.join(associationTableFolderPath, "associations_table.tsv")
+
+associationTableLines = []
+
+with open(associationsTablePath, "r", encoding="utf-8") as associationTable:
+    for line in associationTable:
+        associationTableLines.append(unicodedata.normalize("NFKD", line))
```
Comment (mpage21) [inline update_database_scripts/normalizeChars.py:16]: Will you include a comment explaining this line? specifically what NFKD stands for

File: update_database_scripts/filterBetaUnits.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
+from sys import argv
+from os import path
+
+# Studies should have the beta unit for all associations that have the same trait/pValAnno/betaAnno/valType/studyID
+# combination. When they don't (either due to spelling errors in the beta unit column or otherwise), this script
+# removes those associations and prints out the number of associations removed as well as the studies affected.
```
Comment (mpage21) [inline update_database_scripts/filterBetaUnits.py:6]: When you say it removes those associations, you mean that it removes the unique combination right?

File: update_database_scripts/filterBetaUnits.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        lineItems = line.strip().split("\t")
+        id = lineItems[0]
+        trait = lineItems[6]
+        pValueAnnotation = lineItems[11]
+        betaAnnotation = lineItems[17]
+        ogValueTypes = lineItems[18]
+        studyID = lineItems[22]
+        betaUnit = lineItems[16].lower()
+        group = "|".join([trait, pValueAnnotation, betaAnnotation, ogValueTypes, studyID])
+        if group not in uniqueGroups:
```
Comment (mpage21) [inline update_database_scripts/filterBetaUnits.py:35]: Maybe add just a few more comments.

**Review by mpage21 — CHANGES_REQUESTED** [review]
See my comments. Then I think it is good to go!

File: update_database_scripts/filterBetaUnits.py (hunk 2 above)
Comment (MattCloward) [reply update_database_scripts/filterBetaUnits.py:6]: I mean it removes all associations from that combination. Is that correct?

Comment (mpage21) [reply update_database_scripts/filterBetaUnits.py:6]: Yes :)

**Review by mpage21 — APPROVED** [review]

**Pull request merged**